src/Modules/Data/Commands.Data/help/Add-PowerBIDataset.md

```diff
@@ -0,0 +1,126 @@
+---
+external help file: Microsoft.PowerBI.Commands.Data.dll-Help.xml
+Module Name: MicrosoftPowerBIMgmt.Data
+online version:
+schema: 2.0.0
+---
+
+# Add-PowerBIDataset
+
+## SYNOPSIS
+Creates new dataset to Power BI.
+
+## SYNTAX
+
+### Dataset (Default)
+```
+Add-PowerBIDataset -Dataset <Dataset> [-WorkspaceId <Guid>] [-Scope <PowerBIUserScope>] [<CommonParameters>]
+```
+
+### Workspace
+```
+Add-PowerBIDataset -Dataset <Dataset> -Workspace <Workspace> [-Scope <PowerBIUserScope>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Add-PowerBIDataset creates new push dataset to PowerBI.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\>$col1 = New-PowerBIColumn -Name ID -DataType Int64
+PS C:\>$col2 = New-PowerBIColumn -Name Data -DataType String
+PS C:\>$table1 = New-PowerBITable -Name SampleTable1 -Columns $col1,$col2
+PS C:\>
+PS C:\>$col3 = New-PowerBIColumn -Name ID -DataType Int64
+PS C:\>$col4 = New-PowerBIColumn -Name Date -DataType DateTime
+PS C:\>$col5 = New-PowerBIColumn -Name Detail -DataType String
+PS C:\>$col6 = New-PowerBIColumn -Name Result -DataType Double
+PS C:\>$table2 = New-PowerBITable -Name SampleTable2 -Columns $col3,$col4,$col5,$col6
+PS C:\>
+PS C:\>$dataset = New-PowerBIDataSet -Name SampleDataSet -Tables $table1,$table2
+PS C:\>
+PS C:\>Add-PowerBIDataSet -DataSet $dataset
+```
+
+This example instantiate a table with two columns and another table with four columns, and instantiate a dataset.
+Then, it creates the dataset in Power BI.
+
+## PARAMETERS
+
+### -Dataset
+The dataset to be added.
+
+```yaml
+Type: Dataset
+Parameter Sets: (All)
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Scope
```

Comment:
We can remove -Scope as we don't support Admin API.
When you remove it from C# class, the build process should remove this parameter for you from the markdown.

Reply:
Okay, but we had discussion before to put it in even though we dont support it yet. Please let me know which we really want. I don't have any issue removing it though.

Reply:
For now I we should remove it. For Get cmdlets we can assume to always add scope.